memmap_test.go

```diff
@@ -449,3 +449,47 @@ func TestMemFsUnexpectedEOF(t *testing.T) {
 		t.Fatal("Expected ErrUnexpectedEOF")
 	}
 }
+
+func TestMemFsRenameDir(t *testing.T) {
+	const srcPath = "/src"
+	const dstPath = "/dst"
+	const subDir = "dir"
+
+	fs := NewMemMapFs()
+
+	err := fs.MkdirAll(srcPath+FilePathSeparator+subDir, 0777)
+	if err != nil {
+		t.Errorf("MkDirAll failed: %s", err)
+		return
+	}
```

Comment:
Can you also add a file in the subdirectory and not have only directories in the test?